ironfish/src/blockchain/blockchain.ts

```diff
@@ -205,21 +244,58 @@ export class Blockchain<
     return Math.max(Math.min(1, progress), 0)
   }
 
+  private loadDefaultGenesisBlock = () => {
+    return Promise.resolve(IJSON.parse(genesisBlockData) as SerializedBlock<SH, ST>)
+  }
+
   async open(): Promise<void> {
     if (this.opened) return
     this.opened = true
 
     await this.db.open()
 
-    this.genesis = await this.getHeaderAtSequence(GENESIS_BLOCK_SEQUENCE)
+    const genesis = await this.getHeaderAtSequence(GENESIS_BLOCK_SEQUENCE)
```

Comment:
Can we move this into a function on the class called seed?

Reply:
👍 pulled this out